.vib/gitea/goss/gitea.yaml

```diff
@@ -0,0 +1,65 @@
+group:
+  gitea:
+    exists: true 
+user:   
+  gitea:
+    exists: true
+file:
+  /opt/bitnami/gitea:     
+    exists: true     
+    mode: "0775"     
+    owner: gitea     
+    group: root
+    filetype: directory
+  /opt/bitnami/gitea/bin/gitea:
+    exists: true
+    mode: "0775"
+    filetype: file
+  /opt/bitnami/gitea/custom:
+    exists: true
+    mode: "0775"
+    owner: gitea
+    group: root
+    filetype: directory
+  /opt/bitnami/gitea/custom/conf:
+    exists: true
+    mode: "0775"
+    owner: gitea
+    group: root
+    filetype: directory
+  /opt/bitnami/gitea/data:
+    exists: true
+    mode: "0775"
+    owner: gitea
+    group: root
+    filetype: directory
+  /opt/bitnami/gitea/tmp:
+    exists: true
+    mode: "0775"
+    owner: gitea
+    group: root
+    filetype: directory
+  /opt/bitnami/gitea/tmp/log:
+    exists: true
+    mode: "0775"
+    owner: gitea
+    group: root
+    filetype: directory
+  /bitnami/gitea:
+    exists: true
+    mode: "0775"
+    owner: gitea
+    group: root
+    filetype: directory
+  /opt/bitnami/gitea/custom/conf/app.ini.template:
+    exists: true
+    mode: "0664"
+    owner: gitea
+    group: root
+    filetype: file
```

Comment:
Given we are checking the `app.ini` file below, I think we can remove this template check.